### PR TITLE
Update changelog, release notes, and version for 0.5.22-beta22 release

### PR DESCRIPTION
**Pull request by alerickson**
# PR Summary
Update version, changelog and release notes for 0.5.22-beta22 release.  

## PR Checklist

- [x] [PR has a meaningful title](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
    - Use the present tense and imperative mood when describing your changes
- [x] [Summarized changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] [Make sure all `.h`, `.cpp`, `.cs`, `.ps1` and `.psm1` files have the correct copyright header](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [ ] This PR is ready to merge and is not [Work in Progress](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---work-in-progress).
    - If the PR is work in progress, please add the prefix `WIP:` or `[ WIP ]` to the beginning of the title (the `WIP` bot will keep its status check at `Pending` while the prefix is present) and remove the prefix when the PR is ready.
- **[Breaking changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#making-breaking-changes)**
    - [x] None
    - **OR**
    - [ ] [Documentation needed]()
        - [ ] Issue filed: 
- **User-facing changes**
    - [x] Not Applicable
    - **OR**
    - [ ] [Documentation needed]()
        - [ ] Issue filed: 
- **Testing - New and feature**
    - [x] N/A or can only be tested interactively
    - **OR**
    - [ ] [Make sure you've added a new test if existing tests do not effectively test the code changed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#before-submitting)
- **Tooling**
    - [x] I have considered the user experience from a tooling perspective and don't believe tooling will be impacted.
    - **OR**
    - [ ] I have considered the user experience from a tooling perspective and enumerated concerns in the summary.

### CODE REVIEW

**Comment by SydneyhSmith** [comment]
@sdwheeler do we need to update the help uri?

**Comment by sdwheeler** [comment]
> @sdwheeler do we need to update the help uri?

Yes, you need two new FWLinks, one for each module, PSResourceGet and the compat module.

The target of both links needs to be "https://pshelpprod.blob.core.windows.net/cabinets/powershellget-3.x/".

Do you want me to create them?

**Comment by SydneyhSmith** [comment]
> > @sdwheeler do we need to update the help uri?
> 
> Yes, you need two new FWLinks, one for each module, PSResourceGet and the compat module.
> 
> The target of both links needs to be "https://pshelpprod.blob.core.windows.net/cabinets/powershellget-3.x/".
> 
> Do you want me to create them?

@sdwheeler that would be great!

**Comment by sdwheeler** [comment]
PSResourceGet - https://go.microsoft.com/fwlink/?linkid=2238183
PowerShellGet (compat) - https://go.microsoft.com/fwlink/?linkid=2238451

**Comment by sdwheeler** [comment]
Here is another copy of the same video - https://www.youtube.com/watch?v=BXmUE5qWmyQ

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,31 @@
 # CHANGELOG
+## 0.5.22-beta22
+
+### Breaking Changes
+- PowerShellGet is now PSResourceGet! (#1164)
+- Update-PSScriptFile is now Update-PSScriptFileInfo (#1140)
+- New-PSScriptFile is now New-PSScriptFileInfo (#1140)
+- Update-ModuleManifest is now Update-PSModuleManifest (#1139)
+- -Tags parameter changed to -Tag in New-PSScriptFile, Update-PSScriptFileInfo, and Update-ModuleManifest (#1123)
+- Change the type of -InputObject from PSResource to PSResource[] for Install-PSResource, Save-PSResource, and Uninstall-PSResource (#1124)
```
Comment (anamnavi) [inline CHANGELOG.md]:
```suggestion
- Change the type of -InputObject from PSResourceInfo to PSResourceInfo[] for Install-PSResource, Save-PSResource, and Uninstall-PSResource (#1124)

```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,4 +1,31 @@
...
+- PowerShellGet is now PSResourceGet! (#1164)
+- Update-PSScriptFile is now Update-PSScriptFileInfo (#1140)
+- New-PSScriptFile is now New-PSScriptFileInfo (#1140)
+- Update-ModuleManifest is now Update-PSModuleManifest (#1139)
+- -Tags parameter changed to -Tag in New-PSScriptFile, Update-PSScriptFileInfo, and Update-ModuleManifest (#1123)
+- Change the type of -InputObject from PSResource to PSResource[] for Install-PSResource, Save-PSResource, and Uninstall-PSResource (#1124)
+- PSModulePath is no longer referenced when searching paths (#1154)
+
+### New Features
+- Support for Azure Artifacts, GitHub Packages, and Artifactory (#1167, #1180)
```
Comment (anamnavi) [inline CHANGELOG.md]:
```suggestion
- Support for Azure Artifacts, GitHub Packages, and Artifactory (#1167, #1180, #1183)

```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,4 +1,31 @@
...
+- Support for floating versions (#1117)
+- Update, Save, and Install with wildcard gets the latest version within specified range (#1117)
+- Add positonal parameter for -Path in Publish-PSResource (#1111)
+- Uninstall-PSResource -WhatIf now shows version and path of package being uninstalled (#1116)
+- Find returns packages from the highest priority repository only (#1155)
+- Bug fix for PSCredentialInfo constructor (#1156)
+- Bug fix for Install-PSResource -NoClobber parameter (#1121)
+- Save-PSResource now searches through all repos when no repo is specified (#1125)
+- Caching for improved performance in Uninstall-PSResource (#1175)
+- Bug fix for parsing package tags from local repository (#1119)
```
Comment (anamnavi) [inline CHANGELOG.md]:
```suggestion
- Bug fix for parsing package tags for packages that only have .nuspec from local repository (#1119) 

```

**Review by anamnavi — CHANGES_REQUESTED** [review]
Tiny changes, but overall looks great!

**Review by anamnavi — APPROVED** [review]

**Pull request merged**